### PR TITLE
chore(deps): cleanup go.mod a little bit

### PR DESCRIPTION
**Pull request by chenrui333**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -54,7 +49,3 @@ require (
 	gopkg.in/yaml.v3 v3.0.0-20200121175148-a6ecf24a6d71
 	gotest.tools v2.2.0+incompatible
 )
-
-replace github.com/docker/docker => github.com/docker/engine v0.0.0-20181106193140-f5749085e9cb
```
Comment (chenrui333) [inline go.mod]: Upgrade docker dep also removing `golang.org/x/sys` resolution issue.

**Comment by chenrui333** [comment]
I will resolve the merging conflict now.

**Comment by chenrui333** [comment]
@cplee ready for review.

**Comment by cplee** [comment]
sorry @chenrui333 ...i made a mess by pushing an update on `master` 🤦‍♂️

I'll stop touching it, if you can fix this PR then i'll merge it

**Comment by chenrui333** [comment]
No worries, I am fixing it now.

**Comment by chenrui333** [comment]
Should be good now.

**Comment by chenrui333** [comment]
Appreciate your lightening fast merge! 💯

**Pull request merged**